photon-client/src/main/scala/com/linkedin/photon/ml/cli/game/training/GameTrainingDriver.scala

```diff
@@ -14,9 +14,7 @@
  */
 package com.linkedin.photon.ml.cli.game.training
 
-import scala.math.log
-
-import breeze.linalg.DenseVector
+import breeze.numerics.log
```

Comment:
What is the difference between breeze.numerics.log vs scala.math.log? Was there a particular reason to change this?

Reply:
I used breeze.numerics.log for the range vectors. Later on I realized that the ranges for different hyper-parameters are fixed. Although it was represented by a (constant) vector. So we can just use scala.math.log.